planning/obstacle_velocity_planner/launch/obstacle_velocity_planner.launch.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- obstacle velocity planner param -->
+  <arg name="obstacle_velocity_planner_param_path" default="$(find-pkg-share obstacle_velocity_planner)/config/obstacle_velocity_planner.param.yaml"/>
+
+  <!-- vehicle info param -->
+  <arg name="vehicle_info_param_path" default="$(find-pkg-share vehicle_info_util)/config/vehicle_info.param.yaml"/>
+
+  <!-- input -->
+  <arg name="input_trajectory" default="/planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/trajectory"/>
```

Comment:
Note: this topic changed
https://github.com/tier4/autoware_launch/commit/bef983f9becb6760547a192cb800de1d0e488383

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/570/commits/cae27e51ebbad376a2f0ffb919ac2957a1132c0a
DONE